pydantic/utils.py

```diff
@@ -118,6 +121,50 @@ def almost_equal_floats(value_1: float, value_2: float, *, delta: float = 1e-8)
     return abs(value_1 - value_2) <= delta
 
 
+def generate_model_signature(
+    init: Callable[..., None], fields: Dict[str, 'ModelField'], config: Type['BaseConfig']
+) -> Signature:
+    """
+    Generate signature for model based on its fields
+    """
+
+    present_params = signature(init).parameters.values()
+    merged_params: OrderedDict[str, Parameter] = OrderedDict()
```

Comment:
```suggestion
    merged_params: OrderedDict[str, Parameter] = {}
```

No need for ordered dict.